docker/prod/Dockerfile.socialhome

```diff
@@ -0,0 +1,34 @@
+FROM python:3
+
+# Ensure pip and setuptools are up to date as well
+# We need a slightly older setuptools due to a bug in pip-tools
+RUN pip install -U pip setuptools==30.4 pip-tools
+
+## # We need Node for Javascript. Install it and then grab the latest version
+RUN curl -sL https://deb.nodesource.com/setup_9.x | bash -
```

Comment:
I'm pretty sure (at least a little while ago!) node 9 didn't work, some third party library breaks. If you're sure all the frontend compiles fine with node 9, we can change the relevant documentation then. Currently it says [6-8](https://socialhome.readthedocs.io/en/latest/install_guides.html#install-node-js) which I have personally tested to work.